preset/src/shortcuts/rate.ts

```diff
@@ -0,0 +1,14 @@
+export const rateShortcuts: Record<string, string> = {
+	// rate
+	'k-rate': 'fsc',
+
+	// disabled
+	'k-rate--disabled': '',
```

Comment:
I think we can add a read-only attribute and then distinguish it from disabled in style

Reply:
> I think we can add a read-only attribute and then distinguish it from disabled in style

Add a `read-only` attribute and use `disabled` configured `voidIcon` and `voidColor`?

Reply:
> > I think we can add a read-only attribute and then distinguish it from disabled in style
> 
> Add a `read-only` attribute and use `disabled` configured `voidIcon` and `voidColor`?

No, what I mean is that if you retain readonly and disabled, then use the two props disabled and readonly. Their functions are the same, they both disable the interaction of the rate component, but the style of disabled can use `opacity` and `not-allowed`